rotkehlchen/chain/evm/decoding/decoder.py

```diff
@@ -175,6 +177,8 @@ def _add_builtin_decoders(self, rules: DecodingRules) -> None:
         normal loading?
         """
         self._add_single_decoder(class_name='Safemultisig', decoder_class=SafemultisigDecoder, rules=rules)  # noqa: E501
+        self._add_single_decoder(class_name='Oneinchv4', decoder_class=Oneinchv4Decoder, rules=rules)  # noqa: E501
```

Comment:
Hmmm we should not have it for all. Only places where 1inch has deployed. Have they deployed in all ofthe chains we support? I don't think so

Reply:
for v5 is deployed in all the chains we support with the same address. Checked it

for v4 is in all the chains we support but for optimism they have a different address. I've move them to every chain and adjusted on optimism